zipline/pipeline/factors/basic.py

```diff
@@ -501,8 +502,7 @@ class PeerCount(SingleInputMixin, CustomFactor):
     **Default Window Length:** 1
     """
     window_length = 1
-    missing_value = -1
-    dtype = int64_dtype
+    missing_value = nan
```

Comment:
missing value of nan is the default for factors with dtype float64 (which is the default dtype for factors), so we shouldn't need this.